perma_web/Pipfile

```diff
@@ -21,7 +21,7 @@ uwsgitop = "*"                              # uWSGI monitoring
 
 
 # databases
-psycopg2 = ">2.7, <2.9"
+psycopg2 = "==2.8.6"
```

Comment:
This change is not strictly necessary; see https://github.com/psycopg/psycopg2/issues/1293